ocs_ci/framework/conf/ocs_version/ocs-4.99.yaml

```diff
@@ -0,0 +1,12 @@
+---
+DEPLOYMENT:
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.99"
+  default_latest_tag: 'latest-4.99'
+  ocs_csv_channel: "stable-4.9"
```

Comment:
Except the ocp_version shouldn't other tags point to 4.99 itself?

Reply:
done